compiler-core/templates/index.css

```diff
@@ -708,3 +733,343 @@ body.theme-dark .pride .blue {
     width: var(--sidebar-toggle-size);
   }
 }
+
+/* Search */
+
+.search {
```

Comment:
The CSS is pretty much the same as just-the-docs'.
I think it could be changed a bit to be better aligned with the current site styling, however I'd appreciate some help with this part :)